### DISCUSSION TITLE
[A/L] (2021/22) Funzioni - Esercizio 28

### DISCUSSION THREAD

**Discussion by matypist** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#268] [Esercizi A-L]
![image](https://user-images.githubusercontent.com/69389461/152317380-605a8d15-fd52-4175-a0ea-81f1e84b5efa.png)

**Comment by mildora00** [comment]
1) V
2) F
3) V

**Comment by LuginiAndrea** [comment]
La prima è vera sempre
La seconda sarebbe vera solo se $f$ è iniettiva, il che non ci è assicurato, quindi la proposizione è da considerare falsa
Dimostriamo che la terza è vera:
$$x \in A \cap B \Longrightarrow x \in A \vee x \in B \Longrightarrow x \in A$$
$$\forall y \in f\left(A \cap B\right) \exists x \in A \cap B \textrm{ t.c. } f\left(x\right) = y \Longrightarrow x \in A \Longrightarrow y \in f\left(A\right) \Longrightarrow f\left(A \cap B\right) \subset f\left(A\right) \Longrightarrow f\left(A \cap B\right) \subset f\left(A\right) \cup f\left(B\right) $$
